src/schema/resolvers/signup_resolver.js

```diff
@@ -0,0 +1,43 @@
+
+import models from '../../models'
+import createToken from '../../auth'
+
+const resolvers = {
+  Mutation: {
+    signup (parent, { data }, context) {
+      const { username, password, lastName, firstName } = data
+      if (!username || !password) return { error: 'No password or username' }
+      const user = models.user.findOrCreate({
+        where: { username: username },
+        defaults: { password: password, lastName: lastName, firstName: firstName }
+      })
+      const jwt = createToken
+      return {
+        user,
+        jwt,
+        error: 'No'
```

Comment:
The 'No' string implies a semantic, so your client should be aware of getting a 'No' as a response and react to it... it's kind of complex. So, it would be better to just not including the `error` field.